testConnectivity.py

```diff
@@ -87,10 +92,12 @@ def scaCommIsGood(amc, maxIter=5, ohMask=0xfff, nOHs=12):
     """
 
     scaCommPassed = False
+    from gempython.utils.registers_uhal import writeRegister
     from reg_utils.reg_interface.common.sca_utils import sca_reset 
     from reg_utils.reg_interface.common.jtag import initJtagRegAddrs
     initJtagRegAddrs()
     for trial in range(0,maxIter):
+        writeRegister(amc,"GEM_AMC.SLOW_CONTROL.SCA.ADC_MONITORING.MONITORING_OFF",0x0)
```

Comment:
Does this need to happen inside the loop?

Reply:
It seems your fork point is not up-to-date with my remote branch; this should be writing `0xfff` as illustrated [here](https://github.com/cms-gem-daq-project/vfatqc-python-scripts/pull/249/files#diff-49db7e1b7f530db9c88c1268571e14eeR100)

Reply:
> Does this need to happen inside the loop?

No it can be done before line 99 but after line 98.